src/report/report_generate.py

```diff
@@ -0,0 +1,106 @@
+"""
+Script generating monthly report
+"""
+
+import report_utils as utils
+import sys
+from report_data import get_report_data
+from docx import Document
+from docx.enum import section, text
+from os import remove
```

Comment:
Imports should be grouped in the following order:
1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.
You should put a blank line between each group of imports.

Reply:
https://peps.python.org/pep-0008/#imports

Reply:
I'll fix it in the current sprint